.circleci/config.yml

```diff
@@ -395,7 +381,11 @@ workflows:
               only:
                 - develop
     jobs:
-      - staging-test-with-rebase
+      - staging-test-with-rebase:
+          filters:
+            branches:
+              ignore:
+                  - /i18n-.*/
```

Comment:
Since the `nightly` workflow runs only on `develop`, I don't think it's necessary to ignore any branches for the `staging-test-with-rebase` job as part of that workflow.